components/prize_pool/wikis/halo/prize_pool_custom.lua

```diff
@@ -0,0 +1,65 @@
+---
+-- @Liquipedia
+-- wiki=halo
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+local TIER_VALUE = {8, 4, 2}
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	args.allGroupsUseWdl = true
+	local prizePool = PrizePool(args):create()
+
+	prizePool:setLpdbInjector(CustomLpdbInjector())
+
+	return prizePool:build()
+end
+
+function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
+	lpdbData.weight = CustomPrizePool.calculateWeight(
+		lpdbData.prizemoney,
+		Variables.varDefault('tournament_liquipediatier'),
+		placement.placeStart
+	)
+
+	lpdbData.publishertier = Variables.varDefault('tournament_hcs_premier', '')
+	lpdbData.extradata.publisherpremier = Variables.varDefault('tournament_publisher_major') and 'true' or ''
```

Comment:
is that actually used somewhere?
why is there hcs_premier and publisher_major?

Reply:
It is used inconsistently by the organizer HCS. It is mentioned at the following site [https://www.halowaypoint.com/news/introduction](https://www.halowaypoint.com/news/introduction)

Reply:
but is it used on the wiki
and if so should they just alias?

Reply:
After a quick look, all that I can find is that lpdbData.publishertier is used to denote if a tournament is HCS sanctioned or not. This is only set in the Match2 table from {{Infobox league|}} and not the Placement table. There's nothing in extra data. I am not sure if this would be for additional functionality or left over from a fork. That's a question for @FenrirWulf

Reply:
Couldnt tell you tbh. I am just putting this on Github for Kano. I have no issues removing it